lib/servers/stepper_motor_server/stepper_motor_firmware/stepper_motor_firmware.ino

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
I would just call the subfolder "firmware", you're already in the stepper_motor_server directory.